on-prem-docs/scaling.md

```diff
@@ -2,27 +2,27 @@
 With any tiered or HA deployment of the builder services you'll likely want to horizontally scale your front-end nodes. The most common deployment pattern for this usecase is a pool of front-end nodes fronted by a load-balancer.
 
 ## Deploying New Front-ends
-The on-prem-builder install.sh script now supports scaling front-end nodes as a deployment pattern. It is require that new front-ends be deployed on a separate compute from your initial on-prem deployment. Similarly to Chef Automate's bootstrap pattern the on-prem builder install script can generate a bootstrap bundle which is used to simplify the deployment of new front-ends.
+The on-prem-builder install.sh script now supports scaling front-end nodes as a deployment pattern. It is required that new front-ends be deployed on a separate compute from your initial on-prem deployment. Since builder services will now need to communicate accross accross your network to the frontend and backend nodes, we recomment that the following ports be open to these nodes in order to guarantee your on-prem builder properly functions:
```

Comment:
Maybe strengthen clause "we recommend that the following ports" to "we require ..."

Reply:
good call. changed.